lib/types/types.ts

```diff
@@ -56,12 +65,14 @@ export interface FieldDefinition {
     optional?: boolean;
     repeated?: boolean;
     fields?: SchemaDefinition;
+    statistics?: Statistics
 }
 
 export interface ParquetField {
     name: string;
     path: string[];
-    key: string;
+    // key: string;
+    statistics?: Statistics
```

Comment:
Temporarily commented out key because it was not being initialized along with the other elements in schema. Statistics type was made to match usage in schema to match

Reply:
Was `key` never even being used?

Reply:
So far I only see ParquetField used in schema.ts. Other than that I don't see it explicitly used after looking for the `key` keyword in some of the files